src/shared/hwdb-util.c

```diff
@@ -607,6 +611,15 @@ int hwdb_update(const char *root, const char *hwdb_bin_dir, bool strict, bool co
         if (err < 0)
                 return log_error_errno(err, "Failed to enumerate hwdb files: %m");
 
+        if (strv_isempty(files)) {
+                if (unlink(hwdb_bin) >= 0)
+                        log_info("No hwdb files found, removing existing hwdb database at %s.", hwdb_bin);
+                else
+                        log_info("NO hwdb files found, skipping");
```

Comment:
NO → No

But maybe actually handle the error, i.e.:

```c
if (unlink(hwdb_bin) < 0) {
        if (errno != ENOENT)
                return log_error_errno(errno, "Failed to remove compiled hwdb database %s: %m", hwdb_bin);

        log_info("No hwdb files found, skipping.");
} else 
        log_info("No hwdb files found, compiled hwdb database %s removed.");

return 0;
```

or something like that?